lib/mini_defender/rules/url.rb

```diff
@@ -1,17 +1,79 @@
 # frozen_string_literal: true
 
 require 'uri'
+require 'resolv'
+require_relative '../detectors/public_domain_detector'
 
 class MiniDefender::Rules::Url < MiniDefender::Rule
+  ALLOWED_MODIFIERS = %w[https_only public no_ip]
+
+  def initialize(modifiers = [])
+    @modifiers = Array(modifiers).map(&:to_s)
+    validate_modifiers! unless @modifiers.empty?
```

Comment:
Can you please avoid using the `if` or `unless` modifiers, instead use full statements as in the example below:

```ruby
def initialize(modifiers = [])
  @modifiers = Array(modifiers).map(&:to_s)

  unless @modifiers.empty?
    validate_modifiers! 
  end
end
```

Reply:
Noted 🌟